src/radical/pilot/pmgr/launching/default.py

```diff
@@ -1281,6 +1282,7 @@ def _prepare_pilot(self, resource, rcfg, pilot, expand):
         jd.error                 = "bootstrap_0.err"
         jd.total_cpu_count       = number_cores
         jd.total_gpu_count       = number_gpus
+        jd.total_physical_memory = required_memory
```

Comment:
SAGA's `total_physical_memory` on the other hand is supposed to define the global job memory.  If the above is changed to global memory too, then this should work - right now it should result in incorrect memory specs on the SAGA layer I think.

Reply:
In SAGA `total_physical_memory` translates into SLURM parameter `--mem` ([slurm](https://slurm.schedmd.com/sbatch.html)), which is real memory required per node

Reply:
Thanks for noting that - but then the SAGA Slurm adaptor needs fixing (see https://www.ogf.org/documents/GFD.90.pdf page 179).

Reply:
Thus we just set a total memory here (no more changes in RP, except removing that comment "per node", right?) and as the next step is to fix SAGA (translates total memory into memory per node for SLURM)?

Reply:
Yes, correct!  Please add a small comment on the new dscription key also, otherwise that looks good.